### PR TITLE
fix: enable usestdlibvars linter

### PR DESCRIPTION
**Pull request by mmorel-35**
enables and fixes [usestdlibvars](https://golangci-lint.run/usage/linters/#usestdlibvars) linter issues

### CODE REVIEW

**Review by ahrtr — APPROVED** [review]

**Review by serathius — APPROVED** [review]

File: tools/etcd-dump-logs/main.go [hunk 1]
```diff
@@ -292,9 +295,9 @@ func printRequest(entry raftpb.Entry) {
 		switch r.Method {
 		case "":
 			fmt.Print("\tnoop")
-		case "SYNC":
+		case MethodSync:
 			fmt.Printf("\tmethod=SYNC time=%q", time.Unix(0, r.Time).UTC())
-		case "QGET", "DELETE":
+		case MethodQGet, MethodDelete:
```
[the remark this answers is not in the document]
Comment (serathius) [reply tools/etcd-dump-logs/main.go:300]: They are methods from etcd v2 API which was removed in main branch. Don't think we should make it part of public API

[the remark this answers is not in the document]
Comment (ahrtr) [reply tools/etcd-dump-logs/main.go:300]: @mmorel-35 please resolve this minor comment (i.e. rename `MethodQGet` to `methodQGet`) before we merge this PR.

**Review by ahrtr — APPROVED** [review]

**Pull request merged**